README.md

```diff
@@ -30,24 +30,38 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+    Map creates a new array. ForEach does not. 
+
 2. What is the difference between a function and a method?
 
+    A function can be called without an object where as a method is a property of an object. 
+
 3. What is closure?
 
+    I'm still not sure, but I think a closure is calling/using something out side of the scope of a function. 
```

Comment:
When you create a function, scope is created. Scope is where the JavaScript lives and is executed. The closure is a collection of all the variables in the scope that were created when the function was created.

When a function gets created and passed around or returned from another function, it carries a backpack with it. And in the backpack are all the variables that were in scope when the function was declared. Closure is the backpack that holds the variables for later.

It took me awhile to get what closure was too. There's a helpful video that I'll send the link to.